tests/tape/test_tape.py

```diff
@@ -1160,15 +1161,15 @@ def test_expand_tape_multiple_wires(self):
 
         assert tape1_exp.graph.hash == tape2.graph.hash
 
-    @pytest.mark.parametrize("ret", [expval, var, sample, counts])
+    @pytest.mark.parametrize("ret", [expval, var, sample, counts, probs])
     def test_expand_tape_multiple_wires_non_commuting(self, ret):
         """Test if a QuantumFunctionError is raised during tape expansion if non-commuting
         observables are on the same wire"""
```

Comment:
Can we update or split this test to cover the specific error messages for the new cases?

Reply:
I agree with @eddddddy .

Reply:
Yup! Just have to list the whole message out 👍🏼

Reply:
Added 👍🏼